src/clj_kondo/impl/linters.clj

```diff
@@ -102,12 +103,26 @@
                                (node->line (:filename ctx) expr :warning
                                            :constant-test-assertion "Test assertion with only constants.")))))
 
+(defn lint-single-key-in [ctx called-name call]
+  (let [keys (-> (:children call)
```

Comment:
It's often good to shortcut linters in case the configuration is set to :off. 
For this to work: 
- Add a default config in impl/config.clj for this linter and set the level to `:off`.
- Check at the start of the linter if the setting is not `identical?` to `:off` and then execute the linter. This saves performance when people do not use the linter.